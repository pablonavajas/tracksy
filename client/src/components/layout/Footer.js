import React, { Fragment, useContext } from 'react';
import { Link } from 'react-router-dom';

const Footer = () => {
  return (
    // <footer className="footer-class section blue white-text center page-footer">
    //   <div className="container">
    //     <p>Tracksy Copyright &copy; 2020</p>
    //   </div>
    // </footer>

    <footer className="page-footer blue">
      <div className="container">
        <div className="row">
          <div className="col l6 s12">
            <h5 className="white-text">Footer Content</h5>
            <p className="grey-text text-lighten-4">
              You can use rows and columns here to organize your footer content.
            </p>
          </div>
        </div>
      </div>
      <div className="footer-copyright">
        <div className="container">Â© 2020 Tracksy Copyright</div>
      </div>
    </footer>
  );
};

export default Footer;
