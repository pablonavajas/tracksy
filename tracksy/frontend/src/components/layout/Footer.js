import React from "react";

const Footer = () => {
  return (
    <div>
      <h1>hehe</h1>
    </div>
  );
};

// const Footer = () => {
//   return (
//     <footer className="page-footer blue">
//       <div className="container">
//         <div className="row">
//           <div className="col l6 s12">
//             <h5 className="white-text">Tracksy</h5>
//             <p className="grey-text text-lighten-4">Relationship Management.</p>
//           </div>
//         </div>
//       </div>
//       <div className="footer-copyright">
//         <div className="container">Â© 2020 Tracksy Copyright</div>
//       </div>
//     </footer>
//   );
// };

export default Footer;
