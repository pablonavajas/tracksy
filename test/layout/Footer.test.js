import React from "react";
import { shallow, mount } from "enzyme";
import Footer from "../../tracksy/frontend/src/components/layout/Footer";
import toJson from "enzyme-to-json";
import "../setupTests";

describe("Footer", () => {
  it("renders vcLinks, when isAuthenticated is true, and isStartup is false", () => {
    const wrapper = shallow(<Footer />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
