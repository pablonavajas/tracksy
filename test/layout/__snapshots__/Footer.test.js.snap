// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders vcLinks, when isAuthenticated is true, and isStartup is false 1`] = `
<footer
  className="page-footer blue"
>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col l6 s12"
      >
        <h5
          className="white-text"
        >
          Tracksy
        </h5>
        <p
          className="grey-text text-lighten-4"
        >
          Relationship Management.
        </p>
      </div>
    </div>
  </div>
  <div
    className="footer-copyright"
  >
    <div
      className="container"
    >
      Â© 2020 Tracksy Copyright
    </div>
  </div>
</footer>
`;
